src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -184,10 +184,12 @@ public override FormulaValue Visit(BinaryOpNode node, SymbolContext context)
                 case BinaryOpKind.EqMedia:
                 case BinaryOpKind.EqBlob:
                 case BinaryOpKind.EqGuid:
+                case BinaryOpKind.EqOptionSetValue:
                     return OperatorBinaryEq(this, context, node.IRContext, args);
 
                 case BinaryOpKind.NeqNumbers:
                 case BinaryOpKind.NeqText:
+                case BinaryOpKind.NeqOptionSetValue:
```

Comment:
any reason for missing NeqTime, NeqDate, etc.?

Reply:
No, weirdly. I noticed this while working through this PR and added some tests that failed due to this. I've filed #148  to track this, and address it in a separate change